### PR TITLE
cdb2_sqlreplay - options to replay from multiple dumps, show diffs vs dump

### PR DESCRIPTION
**Pull request by mponomar**

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Error &#x26A0;.<br>Regression testing: <b>1/410 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>replay_eventlog</p>

File: tools/cdb2_sqlreplay/CMakeLists.txt [hunk 1]
```diff
@@ -18,4 +18,7 @@ target_link_libraries(cdb2_sqlreplay ${libs})
 if(COMDB2_BUILD_STATIC)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libgcc -static-libstdc++")
 endif()
+
+set_property(TARGET cdb2_sqlreplay PROPERTY CXX_STANDARD 11)
```
Comment (nirbhayc) [inline tools/cdb2_sqlreplay/CMakeLists.txt:22]: Wasn't this an issue on IBM and Sun?

File: tools/cdb2_sqlreplay/cdb2_sqlreplay.cpp [hunk 2]
```diff
@@ -492,20 +648,73 @@ void replay(cdb2_hndl_tp *db, cson_value *event_val) {
 
     /* TODO: have switch to print or not results */
```
Comment (nirbhayc) [inline tools/cdb2_sqlreplay/cdb2_sqlreplay.cpp:649]: We now have a switch. :)

**Review by nirbhayc — APPROVED** [review]
Looks good.

File: tools/cdb2_sqlreplay/CMakeLists.txt (hunk 1 above)
Comment (mponomar) [reply tools/cdb2_sqlreplay/CMakeLists.txt:22]: Yes, it was.  For the time being we're only going to make this available on Linux.

**Comment by mponomar** [comment]
Thanks Nirbhay!

**Pull request merged**